Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,26 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
+
+* **GC**. Garbage collector. Garbage collection is an implementation of automatic memory management. .NET currently uses a generational garbage collector, i.e. it groups objects into generations to limit the number of nodes it has to walk for determining which objects are alive. This speeds up collection times.
+
+* **JIT**. Just in time compiler. This technology compiles IL to machine code that the processor understands. It's called JIT because compilation happens on demand and is performed on the same machine the code needs to run on. Since JIT compilation happens during execution of the application, compile time is part of the run time. Thus, JIT compilers have to trade spending more time optimizing code with the savings the resulting code can produce.
+
+* **AOT**. Ahead of time compiler. Similar to JIT this compiler also translates IL to machine code. However, in contrast to a JIT the compilation happens before the application is executed and is usually performed on a different machine. AOT tool chains don't have to trade run time and compile time and thus can spend more time optimizing. Also, since the context of AOT are entire applications, AOT compiler can also perform cross module linking and whole program analysis.
```

Comment:
Comma needed ...

> Similar to JIT, this compiler ...

"However" and "in contrast to" mean the same thing and it presents a little comma (pause) issue after "JIT," so I suggest going from ...

> However, in contrast to a JIT the compilation

... to ...

> In contrast to JIT compilation, AOT compilation ...

Try "occurs" (more definite) in place of "happen" ...

> compilation occurs before

Eliminate "have to" if possible. From ...

> AOT tool chains don't have to trade run time and compile time and thus can spend more time optimizing.

... to perhaps (with "run time" changed to "runtime") ...

> AOT tool chains don't trade runtime for compile time and thus can spend more time optimizing.

I suggest dropping "also" and changing tensing (single compiler, single app) of ...

> Also, since the context of AOT are entire applications, AOT compiler can also perform cross module linking and whole program analysis.

... to ...

> Since the context of AOT is the entire application, the AOT compiler can also perform cross module linking and whole program analysis.

... unless that changes the meaning. It's a little difficult to understand if noting multiple applications is relevant to the meaning of the sentence. If it's important to note multiple applications, then leaving the phrasing close to what you have makes more sense. If the context is just that AOT operates on the single, whole application, then I'd suggest refactoring a hair there. I guess I'm saying ... I'm a little confused here.